src/backend/efiling-reviewer-api/src/test/java/ca/bc/gov/open/jag/efilingreviewerapi/document/documentsApiDelegateImpl/ExtractDocumentFormDataTest.java

```diff
@@ -113,10 +127,10 @@ public void withUserHavingValidRequestShouldReturnCreated() throws IOException,
 
 
     @Test
-    @DisplayName("200: Assert something returned")
-    public void withUserHavingValidRequestShouldReturnCredfated() throws IOException {
+    @DisplayName("Error: Assert something returned")
```

Comment:
Observation: Should the @DisplayName message be changed?

Reply:
Good catch.